src/darwin/Framework/CHIP/MTRDeviceControllerOverXPCParameters.h

```diff
@@ -0,0 +1,25 @@
+/**
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Matter/Matter.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MTRDeviceControllerOverXPCParameters : MTRDeviceControllerAbstractParameters
```

Comment:
Let's come up with a better name for this, I wish I had a better suggestion, but Over is not great. Maybe MTRDeviceControllerXPCParameters?

Reply:
Yep, unattached to the current name so that's as good as any in my view.  Making the change.